scripts/requirements-fixed.txt

```diff
@@ -38,6 +38,7 @@ intelhex==2.3.0
 intervaltree==3.1.0
 isort==5.7.0
 Jinja2==2.11.3
+jsonschema==4.4.0
```

Comment:
Late to the game, but https://github.com/nrfconnect/sdk-nrf-next/pull/1031 brought my attention to this change.

Why is `jsonshema` suddenly placed here.
afact, we don't have any `jsonshema` pip requirement in our regular `requirements-*.txt` files.

Is this requirement only for CI ?
We should know why a given pip package is required, and in this case we try to aling to yaml, so I would really know why json is needed.

Also this file serves as input to NCS Toolchain creation, so we shouldn't hace pip packages described here if they are only needed for CI testing.
If it is required for general development, we need an entry somewhere in a requirements file.

Reply:
@LuDuda @ArekBalysNordic FYI

Reply:
@tejlmand as this requirement is not "only for CI", and it is to update the build system environment for Matter sample accessories, how exactly should I correct it, or who can decide on this?